src/config.c

```diff
@@ -23,11 +25,14 @@ int ReadConfig(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         const char *policy_cstr;
         size_t len;
 
-        if (RMUtil_ParseArgsAfter("COMPACTION_POLICY", argv, argc, "s", &policy) != REDISMODULE_OK) {
+        if (RMUtil_ParseArgsAfter("COMPACTION_POLICY", argv, argc, "s", &policy) !=
+            REDISMODULE_OK) {
             return TSDB_ERROR;
         }
         policy_cstr = RedisModule_StringPtrLen(policy, &len);
-        if (ParseCompactionPolicy(policy_cstr, &TSGlobalConfig.compactionRules, &TSGlobalConfig.compactionRulesCount) != TRUE ) {
+        if (ParseCompactionPolicy(policy_cstr,
```

Comment:
IMHO this is better (+2 indent on if-expr, rather than the python-like indent-to-bracket).
```
ParseCompactionPolicy(policy_cstr, &TSGlobalConfig.compactionRules,
        &TSGlobalConfig.compactionRulesCount) != TRUE) {
    return TSDB_ERROR;
}
```